modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java

```diff
@@ -230,6 +234,7 @@ protected void initModels() {
 		setDDMFormValuesDefaultLocale();
 		setDDMFormValuesAvailableLocales();
 
+		setDDMFormInstanceId();
 		setGroupId();
```

Comment:
@brunofariasdeo, are we able to remove the the blank lines of the method `initModels` and also sort the methods invocations alphabetically?

Reply:
Eagle eyes! I've changed it here :)